tests/integration/sql/test_sql.py

```diff
@@ -275,7 +275,6 @@ def test_sql_read_without_schema_inference(test_db, generated_data) -> None:
         "bool_col": daft.DataType.bool(),
         "date_col": daft.DataType.date(),
         "date_time_col": daft.DataType.timestamp(timeunit="ns"),
-        "time_col": daft.DataType.time(timeunit="ns"),
```

Comment:
Reading with sql alchemy infers this as `Duration`, which could be casted to a `Time`. But this PR disables that behavior, instead you would have to cast to i64, then Time.